journal-letters/modernized/03-epistles.adoc

```diff
@@ -95,11 +100,16 @@ So remain your friends and brethren in the truth, letting you know that we are w
 with love to you all, sending these lines as a token thereof.
 Farewell.
 
+[.signed-section-signature]
 Thomas Wilson, James Dickinson.
 
-=== An Epistle to Friends in Pennsylvania.
+[.chapter-subtitle--blurb]
```

Comment:
I think I told you wrong on this.  Sorry!!! I think this the remaining letter headings in this chapter should be just `[.blurb]` tags, and not `[.chapter-subtitle-blurb]`